devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -37,20 +40,26 @@ stages:
           dockerPath: '**/mavenpipe.dockerfile'
 
   - stage: Deploy
+    condition: >
+      and( succeeded(), 
+      in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'), 
+      in(variables['Build.SourceBranch'], '${{parameters.branchMain}}', '${{parameters.branchDev}}' ) )
```

Comment:
syntax consistency, spacing after {{ and before }} (i won't block it for this though)